jquery.dynatable.js

```diff
@@ -787,7 +792,7 @@
 
     this.position = function() {
       var $processing = $('#dynatable-processing-' + obj.element.id),
-          $span = $processing.children('span'),
+          $span = $processing.find('>:first-child'),
```

Comment:
Is this selector change going to cause issues for calculating the total height of the processing indicator, if someone changes the template to be multiple sibling elements on the top level? E.g., if I call dynatable with `processingTemplate: '<div>{text}</div><div>Please hold on</div>`, this selector will only grab the first div. Maybe we want to change this line to be:

```js
$span = $processing.children()
```

?